packages/wallet-sdk/src/sign/SignRequestHandler.ts

```diff
@@ -4,16 +4,13 @@ import { SignRequestHandlerListener } from './UpdateListenerInterface';
 import { CB_KEYS_URL } from ':core/constants';
 import { standardErrorCodes, standardErrors } from ':core/error';
 import { AddressString } from ':core/type';
-import { RequestArguments } from ':core/type/ProviderInterface';
+import { ConstructorOptions, RequestArguments } from ':core/type/ProviderInterface';
 import { RequestHandler } from ':core/type/RequestHandlerInterface';
 
-interface SignRequestHandlerOptions {
-  appName: string;
-  appLogoUrl?: string | null;
-  appChainIds: number[];
-  smartWalletOnly: boolean;
+type SignRequestHandlerOptions = ConstructorOptions & {
   updateListener: SignRequestHandlerListener;
-}
+  keysUrl?: string;
```

Comment:
wonder what would we use this for? 
also nit: i dont have better names in mind, but think keysUrl might not be really intuitive